standard/lpdb.lua

```diff
@@ -0,0 +1,66 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Lpdb
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lpdb = {}
+
+-- Executes a mass query.
+--[==[
+Loops Lpdb queries to e.g.
+- circumvent the maximum limit of 5000
+- use additional filtering (e.g. because lpdb does not support it)
```

Comment:
```suggestion
Loops LPDB queries to e.g.
- circumvent the maximum limit of 5000
- use additional filtering (e.g. because LPDB does not support it)
```